css-brochure-form/styles.css

```diff
@@ -0,0 +1,74 @@
+body {
+  line-height: 1em;
+  width: 90%;
+  margin: 0 auto;
+}
+
+h3 {
+  color: #ec5d57;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  text-transform: uppercase;
+  position: relative;
+  left: 150px;
+  letter-spacing: 0.1em;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: rgb(45 45 45);
+  margin: 1em;
+  line-height: 1.5em;
+}
+
+input {
+  font-family: AtlasGrotesk, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  padding: 0.6rem;
+  width: 425px;
+  margin-top: 0.1rem;
+  margin-bottom: 1rem;
+  border-color: #cacaca;
+  border-style: solid;
+  border-width: 1.5px;
+  border-radius: 5px;
+}
+
+select {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  padding: 0.5rem;
+  width: 425px;
+  margin-top: 0.1rem;
+  margin-bottom: 0.5rem;
+  background-color: #f8f9fa;
+  border-color: #cacaca;
+  border-style: solid;
+  border-width: 1.5px;
+  border-radius: 5px;
+}
+
+.button {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 14.5px;
+  padding-top: 0.4rem;
+  padding-bottom: 0.45rem;
+  background-color: #ec5d57;
+  color: white;
+  border-radius: 40px;
+  border: 0;
+  width: 95px;
+  position: relative;
+  left: 330px;
```

Comment:
Instead of using a really huge box offset (`left`) here, you can either use `float` or put the button in a `div` with its `text-align` set to `right`.

Reply:
![image](https://user-images.githubusercontent.com/103452411/186540570-778672a1-6659-47f9-8eac-83a22de023de.png)

I used float instead, but since the other elements are written in pixels, I added margin-right so the button would be closer like the picture. However, if I make the screen larger, the button moves in relation to the screen. This was what I came up with without using flex-box. Please let me know if I should edit/change anything to make it better. Thank you!
![image](https://user-images.githubusercontent.com/103452411/186540842-29a62f66-815e-48bf-9580-51e452ab1ea2.png)
![image](https://user-images.githubusercontent.com/103452411/186540889-4f59f6f1-c5cc-4cab-b874-b12a2c28ffb7.png)